Content.Client/Chat/StoredChatMessage.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections.Generic;
+using Content.Client.Interfaces.Chat;
+using Content.Shared.Chat;
+using Robust.Client.Console;
+using Robust.Shared.Interfaces.Network;
+using Robust.Shared.IoC;
+using Robust.Shared.Log;
+using Robust.Shared.Maths;
+using Robust.Shared.Utility;
+using Robust.Client.UserInterface.Controls;
+
+namespace Content.Client.Chat
+{
+    public class StoredChatMessage
+    {
+        // TODO Make me reflected with respect to MsgChatMessage
+
+        /// <summary>
+        ///     Client's own copies of chat messages used in filtering locally
+        /// </summary> 
+
+        /// <summary>
+        ///     Actual Message contents, i.e. words
+        /// </summary>
+        public string Message { get; set; }
+
+        /// <summary>
+        ///     Message channel, used for filtering
+        /// </summary>
+        public ChatChannel Channel { get; set; }
+
+        /// <summary> 
+        ///     What to "wrap" the message contents with. Example is stuff like 'Joe says: "{0}"'
+        /// </summary> 
+        public string MessageWrap { get; set; }
```

Comment:
I think you could actually get away with not storing the message wrap in these, and just having the `Message` be the full line displayed in chat.

The message wrap was mostly for things like speech bubbles. It shouldn't apply here.

Reply:
I'll go ahead and remove it then, didn't know if it would effect emotes and the like.